app-backend/db/src/main/scala/ObjectPermissions.scala

```diff
@@ -192,18 +193,23 @@ trait ObjectPermissions[Model] {
                     actionType: ActionType,
                     ownershipTypeO: Option[String] = None,
                     groupTypeO: Option[GroupType] = None,
-                    groupIdO: Option[UUID] = None): Option[Fragment] = {
+                    groupIdO: Option[UUID] = None,
+                    tableNameO: Option[String] = None): Option[Fragment] = {
+    val tableName: String = tableNameO match {
+      case Some(tableName) => s"${tableName}."
+      case _ => ""
+    }
     val ownedF: Fragment =
-      Fragment.const(s"owner = '${user.id}'")
+      Fragment.const(s"${tableName}owner = '${user.id}'")
```

Comment:
is `Fragment.const` required here to prevent `tableName` from being quoted?
It might be safer to do something like `Fragment.const(s"${tableName}owner = ") ++ fr"${user.id}")

Reply:
Yeah, I agree that making the contents of any `Fragment.const`s as small as possible is desirable.

Reply:
`Fragment.const(tableName) ++`...